docs/source/api/entities.md

```diff
@@ -0,0 +1,329 @@
+      },
+      "plan": {
+        "billingInterval": "daily",
+        "billingIntervalCount": 1,
+        "usageCalculationType": "metered",
+        "trialInterval": "daily",
+        "trialIntervalCount": 1
+      },
+      "bundleItems": [
+        {
+          "productId": "c",
+          "quantity": 1,
+          "configuration": [
+            {
+              "key": "greeting",
+              "value": "For my Darling"
+            }
+          ]
+        }
+      ],
+      "meta": {},
+      "content": {
+        "de": {
+          "created": null,
+          "updated": null,
+          "vendor": "Herstellername",
+          "brand": "Marke",
+          "title": "Produktname",
+          "slug": "produktname",
+          "subtitle": "Short description",
+          "description": "Long description",
+          "labels": [
+            "Neu"
+          ]
+        }
+      }
+    },
+    "media": [
+      {
+        "_id": null,
+        "created": null,
+        "updated": null,
+        "asset": {
+          "_id": null,
+          "contentHash": null,
+          "url": "https://www.story.one/media/images/poop-4108423_1920.width-1600.format-jpeg.jpg"
+        },
+        "tags": ["big"],
+        "meta": {},
+        "content": {
+          "de": {
+            "created": null,
+            "updated": null,
+            "title": "Produktname",
+            "subtitle": "Short description",
+          }
+        }
+      }
+    ],
+    "variations": [
+      {
+        "_id": null,
+        "created": null,
+        "updated": null,
+        "key": "color",
+        "type": "COLOR",
+        "options": [
+          {
+            "value": "ff0000",
+            "content": {
+              "de": {
+                "created": null,
+                "updated": null,
+                "title": "Rot",
+                "subtitle": ""
+              }
+            }
+          }
+        ],
+        "content": {
+          "de": {
+            "created": null,
+            "updated": null,
+            "title": "Farbe",
+            "subtitle": "Farbvariante"
+          }
+        }
+      }
+    ],
+    "reviews": [
+      {
+        "_id": null,
+        "created": null,
+        "updated": null,
+        "authorId": "root",
+        "rating": 1,
+        "title": "What the?",
+        "review": "That product just sucks big times",
+        "votes": [
+          {
+            "timestamp": null,
+            "userId": "root",
+            "type": "UPVOTE",
+            "meta": {}
+          }
+        ],
+        "meta": {}
+      }
+    ]
+  }
+}
+```
+
+### Entity Type: Assortment
+
+Set by unchained:
+- authorId,
+- slug history
+- _id, created & updated if not provided
+
+```json
+{
+  "entity": "ASSORTMENT",
+  "operation": "CREATE",
+  "payload": {
+    "specification": {
+      "_id": "A",
+      "created": null,
+      "updated": null,
+      "sequence": "",
+      "isActive": true,
+      "isBase": false,
+      "isRoot": true,
```

Comment:
Difference between isBase and isRoot is not obvious for me.

Reply:
I once wanted to change the default behavior of Query.assortment in a way that when you did not specify assortmentId it will return the base assortment. This would solve some issues we had at different clients where we had include/exclude certain assortments when just going for the isRoot ones.

Reply:
what do you think?

Reply:
Hmm, I don't get it. Can you elaborate the case more, where you want to have a `isBase: false` but `isRoot: true`? Maybe it's a good learn for others too.

Reply:
isRoot assortments in fictional shop: Public Products (isBase = true), Mandate Y Products, Mandate X Products
Public Products child assortments: Men, Women, Children

Requirement: Only show public products